Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,34 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+
+  depends_on "doxygen" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "docs"
+    system "make", "install"
+
+    # compiler shims don't seem to work after install
+    ## Is there a better way that would get a "regular" full path the compilers
+    ## that are used to build the package?
+    inreplace "#{lib}/kim-api-v2/cmake/kim-api-v2-config.cmake" do |s|
```

Comment:
Can't you specify this to `cmake` as argument?

Reply:
The package "remembers" which compilers were used to build it so that later "plugins" can be built with the same compilers.  Otherwise things won't work correctly.

Reply:
It's not the only software we ship that does that, but there usually is a way to override what it detects.